index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Survey Form</title>
   <link rel="stylesheet" href="index.css">
 </head>
+
 <body>
-  
+  <div class="main-content">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
```

Comment:
Use semantic HTML tags. For example, here I would advise you to use the <header></header> tag. These will improve the web accessibility of your app.

Reply:
- [ ] Well done on making the review comment. It would have been much better if your review comment is clear on what should be done. Also, it sounds more like a command or order. Usually, when giving code reviews, it is expected that your points be clear and easy to understand and that it is polite. 

> Use semantic HTML tags. For example, here I would advise you to use the tag. These will improve the web accessibility of your app.

The first sentence is ambiguous, and not clear. The question is **_what tag are you talking about?_.** Kindly state clearly what and why it should be done. **_Your comment above was good in that it gave a reason for the suggestion._**